api/base/serializers.py

```diff
@@ -750,7 +753,10 @@ def get_url(self, obj, view_name, request, format):
                     urls[view_name] = {}
                 else:
                     if callable(view):
-                        view = view(getattr(obj, self.field_name))
+                        if self.view_lambda_argument:
+                            view = view(getattr(obj, self.view_lambda_argument))
```

Comment:
rather than adding an attribute to the superclass that's only set by the subclass, maybe add a "_handle_callable_view" function to the superclass and have the subclass implement the logic to leverage the `view_lambda_argument`?